rest_framework/versioning.py

```diff
@@ -74,7 +74,17 @@ class URLPathVersioning(BaseVersioning):
     invalid_version_message = _('Invalid version in URL path.')
 
     def determine_version(self, request, *args, **kwargs):
+        """
+        Use Default Version If 'version' Did Not Sets by Client in URL (So, It's Equal to None).
+        e.g:
+        api/users/ > Use Default Version (As Often The Default Version Is The Latest One).
+        api/v1/users/ > Check 'v1' Is Allowed?
+        """
         version = kwargs.get(self.version_param, self.default_version)
+
+        if version is None:
+            version = self.default_version
```

Comment:
I think it'd be marginally cleaner as:

```python
version = kwargs.get(self.version_param)
if version is None:
    version = self.default_version
```

Note that we're already checking for "version not in the regex groups", but we're not checking for "version is in the regex groups, but is None".